frontend/src/components/ShootDetails/ShootKubeconfig.vue

```diff
@@ -88,17 +100,29 @@ export default {
   },
   data () {
     return {
-      kubeconfigExpansionPanel: false
+      kubeconfigExpansionPanel: false,
+      adminKubeConfigExpiration: '30m', // Default 30 minutes
+      adminKubeConfig: undefined
```

Comment:
remove this data. we do not want to cache it (as explained below)

Reply:
The current kubeconfig needs to be cached for the computed "kubeconfig" property though or am I missing something?
We can't fetch the admin kubeconfig async from within the computed property.

I changed the behaviour, that a new kubeconfig is always fetched.